packages/electron-mksnapshot/README.md

```diff
@@ -0,0 +1,27 @@
+## @packages/electron-mksnapshot
```

Comment:
Should this be baked into the binary code or should this be in located under `npm`?

Reply:
Had an offline discussion. We decided it made sense to move a lot of this new development only code to a `tooling` namespace

Reply:
@ryanthemanuel Love the new workspace name!